.changeset/shaggy-feet-live.md

```diff
@@ -0,0 +1,9 @@
+---
+"@comet/brevo-admin": major
+---
+
+Make this package compatible with [COMET v6](https://docs.comet-dxp.com/docs/migration/migration-from-v5-to-v6)
+
+**Breaking Changes**:
+
+- Now requires >= v6.0.0 for `@comet` packages
```

Comment:
The file can be deleted, right?
Everything is in the other changeset file as well

Reply:
This changeset is for `brevo-admin`, the other is for `brevo-api`. Both only contain the relevant info for this package, so I'd keep both

The alternative would be to share one changeset for both packages. But then the changeset is copied twice to the resulting changelog and the `brevo-admin` section contains irrelevant info.

Reply:
>  This changeset is for brevo-admin, the other is for brevo-api

👍